cranelift/codegen/src/isa/pulley_shared/lower.isle

```diff
@@ -331,11 +331,36 @@
   (pulley_xmul64_s8 a b))
 
 ;; 128-bit (or wide) multiplication
-(rule (lower (has_type $I128 (imul (uextend a) (uextend b))))
+(rule 4 (lower (has_type $I128 (imul (uextend a) (uextend b))))
   (pulley_xwidemul64_u (zext64 a) (zext64 b)))
-(rule (lower (has_type $I128 (imul (sextend a) (sextend b))))
+(rule 4 (lower (has_type $I128 (imul (sextend a) (sextend b))))
   (pulley_xwidemul64_s (sext64 a) (sext64 b)))
 
+;; for I128
+(rule (lower (has_type $I128 (imul x y)))
+  (let
+    ((x_regs ValueRegs x)
+      (x_lo XReg (value_regs_get x_regs 0))
```

Comment:
indented one space too far here? (sorry, minor issue, my Lisp sensibilities are just causing an eyeball twitch)

Reply:
Currently ISLE backends differ a bit on style where the x64 backend likes to line things up by the previous line in ISLE (I think the style you're expecting here) but the riscv64 backend takes a different approach where the next line is just indented by one more than the previous line at all times (which is the style I was writing here). Personally I prefer the riscv64 style because I find it easier to work with editors and I don't have to think too hard about lining things up all the time. 

That being said I don't really have a strong preference either way, but in the absence of a style guide/formatter I'll probably leave this as-is since it doesn't deviate too far from the current style (AFAIK)

Reply:
If you prefer a different bikeshed color then don't take my comment too seriously -- in the fullness of time maybe we'll have an autoformatter (and then everything will be in the One True S-expr Indent Style) but no worries otherwise :-)